WalletWasabi.Daemon/Rpc/WasabiJsonRpcService.cs

```diff
@@ -169,6 +145,51 @@ public JsonRpcResult WalletInfo()
 		return info;
 	}
 
+	[JsonRpcMethod("getcoinjoininfo")]
+	public JsonRpcResult GetCoinjoinInfo(string? password = null)
+	{
+		var coinJoinManager = Global.HostedServices.Get<CoinJoinManager>();
+		var activeWallet = Guard.NotNull(nameof(ActiveWallet), ActiveWallet);
+		var serverTipHeight = activeWallet.BitcoinStore.SmartHeaderChain.ServerTipHeight;
+
+		AssertWalletIsLoaded();
+		AssertWalletIsLoggedIn(activeWallet, password ?? "");
```

Comment:
Don't need to validate password for null, because it will be validated later inside [Guard.cs](https://github.com/turbolay/WalletWasabi/blob/5c736eca576dbe30488431c457b4cea9a8a75b7f/WalletWasabi/Helpers/Guard.cs#L164)